database/models/fans.js

```diff
@@ -0,0 +1,132 @@
+const client = require('../index');
+
+module.exports = {
+  get: function(req, res) {
+    client.query(`
+      SELECT json_build_object(
+       'username', (SELECT username FROM authentication a WHERE a.id = f.authentication_id),
+       'address', (SELECT
+          json_build_object(
+            'city', f.city,
+            'state', f.state,
+            'street', f.street
+          )
+          FROM fans f
+          WHERE f.id = '${req.params.fanId}'
+        ),
+        'events', (SELECT
+          array_to_json(
+            array_agg(
+              json_build_object(
+                'event_id', e.id,
+                'event_name', e.name,
+                'event_street', e.street,
+                'event_city', e.city,
+                'event_state', e.state,
+                'event_longitude', e.longitude,
+                'event_latitude', e.latitude,
+                'event_timestamp', e.timestamp,
+                'event_start', e.start,
```

Comment:
'start' at line 29 and 'end' at line 30 is named 'start_time' and 'end_time' in event table